src/router/validation/hapiFhirLambdaValidator.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { InvalidResourceError, Validator } from 'fhir-works-on-aws-interface';
+import { Lambda } from 'aws-sdk';
+import AWS from '../../AWS';
+
+interface ErrorMessage {
+    severity: string;
+    msg: string;
+}
+
+interface HapiValidatorResponse {
+    errorMessages: ErrorMessage[];
+    successful: boolean;
+}
+
+export default class HapiFhirLambdaValidator implements Validator {
```

Comment:
Just to confirm this is being added but not enabled by default?

Reply:
It is not enabled by default.

To enable it it needs to be added to FhirConfig: https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/src/config.ts#L75